macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -195,6 +203,10 @@ class TestButtonViewController: NSViewController, NSMenuDelegate {
 			NSTextField(labelWithString: "Large Secondary"),
 			NSTextField(labelWithString: "Large Acrylic"),
 			NSTextField(labelWithString: "Large Borderless"),
+			NSTextField(labelWithString: "Medium (tighter-spacing) Primary"),
```

Comment:
what does `tighter-spacing` mean to convey? Since all Medium's have it can we just omit it for brevity?

Reply:
Yeah. It was thought that the tighter spacing was making the buttons look no larger than the "small" size, but that's not the case as the test app is showing them smaller for some reason (see above comment). So once that's resolved we can remove the "tighter-spacing" label.